Directory.Build.props

```diff
@@ -58,15 +54,11 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="5.0.3" />
-        <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.11.0" />
-        <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.2" />
-        <PackageReference Include="Microsoft.VisualStudio.Threading.Analyzers" Version="16.10.56" />
+        <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="6.0.0" />
```

Comment:
I think I'd remove this one - per the recommendation in the note [here](https://docs.microsoft.com/en-us/visualstudio/code-quality/install-net-analyzers?view=vs-2022).  It's included and enabled by default in .NET 5+.

Reply:
removed.